### PR TITLE
Newsletter 258 translate in french

### PR DESCRIPTION
**Pull request by Copinmalin**

### CODE REVIEW

**Review by Jluc-bitcoinfr — APPROVED** [review]

**Comment by murchandamus** [comment]
I have the impression that this fails because it’s linking to the 7th Mempool series post, but that file is not present in this build. So, it depends on #1182 getting merged

File: _posts/fr/newsletters/2023-07-05-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+_Aucune nouvelle significative n'a été trouvée cette semaine dans les listes de diffusion Bitcoin-Dev et Lightning-Dev._
+
+## En attente de confirmation #8 : La politique comme interface
+
+_Une [série hebdomadaire limitée][policy series] sur le relais des transactions, l'inclusion dans le mempool et la sélection
+des transactions minières---y compris pourquoi Bitcoin Core a une politique plus restrictive que celle autorisée par consensus
+et comment les portefeuilles peuvent utiliser cette politique de la manière la plus efficace._
+
+{% include specials/policy/en/08-interface.md %} {% assign timestamp="0:30" %}
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2023-07-05-newsletter.md]:
```suggestion
{% include specials/policy/fr/08-interface.md %} {% assign timestamp="0:30" %}
```

File: _posts/fr/2023-05-17-policy.md [hunk 2]
```diff
@@ -63,7 +63,19 @@ h2:not(:first-of-type) { margin-top: 3em; }
...
 
+## Ressources du réseau
+
+*Originally published in [Newsletter #257](/fr/newsletters/2023/06/28/#en-attente-de-confirmation-7--ressources-du-réseau)*
+
+{% include specials/policy/fr/07-ressources-du-reseau.md %}
+
+## La politique comme interface
+
+*Originally published in [Newsletter #258](/fr/newsletters/2023/07/05/#en-attente-de-confirmation-8--la-politique-comme-interface)*
```
Comment (bitschmidty) [inline _posts/fr/2023-05-17-policy.md:74]: We have "Originally published in Newsletter..." for the policy entries already (sorry I missed this). But we should consider a follow up PR to translate those.

**Review by bitschmidty — COMMENTED** [review]
Pushed a commit to change a link from /en/ to /fr/ equivalent.

Rebased on master to pull in the 257 references.

Built and tested links for 258

**Pull request merged**